lib/cn/cn_tree.c

```diff
@@ -933,17 +940,6 @@ cn_comp_release(struct cn_compaction_work *w)
     }
 
     perfc_inc(w->cw_pc, PERFC_BA_CNCOMP_FINISH);
-
-    if (HSE_UNLIKELY(!w->cw_completion)) {
-        free(w);
-        return;
-    }
-
-    /* After this function returns the job will be disassociated
-     * from its thread and hence becomes a zombie.  Do not touch
-     * *w afterward as it may have already been freed.
-     */
-    w->cw_completion(w);
```

Comment:
Search for cw_completion and get rid of it (in cn_tree.c).

Reply:
There are no instances of that anymore.

Reply:
No, but there are comments that mention it...